src/config/oauth.json

```diff
@@ -4,6 +4,10 @@
   "api_server_base_path": "{{ .Env.K8S_API_SERVER_BASE_PATH }}",
   "ws_api_server": "{{ .Env.WS_K8S_API_SERVER }}",
   "openshift_console_url": "{{ .Env.OPENSHIFT_CONSOLE_URL }}",
+  "forge_api_url": "{{ .Env.FABRIC8_FORGE_API_URL }}",
```

Comment:
BTW the initial reason for fabric8-runtime-console was for something upstream if we didn't go with all the planner stuff; but now am thinking its gonna be easier just having the same UI both upstream and for openshift.io - so maybe we just merge fabric8-runtime-console into fabric8-ui? We could always split them apart later on to make them more modular (would require moving all the shared common stuff into a separate npm module really first)